docassemble/ALWeaver/data/templates/output.mako

```diff
@@ -9,66 +9,52 @@ include:
   % endfor
 ---
 metadata:
-  title: |
+  title: >-
     ${ interview.title }
-  short title: |
+  short title: >-
     ${ interview.short_title }
+  description: |-
+${ indent(interview.description, by=4) }
   % if interview.categories.any_true():
   tags:
-    % for category in interview.categories.true_values():
-    - ${ category }
+    % for category in set(interview.categories.true_values()) - {'Other'}:
+    - "${ escape_double_quoted_yaml(oneline(category)) }"
+    % endfor
+    % if interview.categories['Other']:
+    % for category in interview.other_categories.split(','):
+    - "${ escape_double_quoted_yaml(oneline(category.strip())) }"
     % endfor
+    % endif
+  % else:
+  tags: []
   % endif
   authors:
     % for author in interview.author.splitlines():
     - ${ author }
     % endfor
----
-mandatory: True
-comment: |
-  Global interview metadata
-variable name: interview_metadata["${ interview.interview_label }"]
-data:
-  al_weaver_version: "${ package_version_number }"
-  generated on: "${ today().format("yyyy-MM-dd") }"
-  title: >-
-    ${ oneline(interview.title) }
-  short title: >-
-    ${ oneline(interview.short_title) }
-  description: |-
-${ indent(interview.description, by=4) }
   % if interview.original_form:
-  original_form: >-
-${ indent(interview.original_form, by=4) }
+  original_form:
+    - ${ interview.original_form }
```

Comment:
Not needed now, but should we starting asking for multiple original forms, one for each PDF / docx that people run through the Weaver?

Reply:
Something to think about, yeah. I think in most cases one URL would work (we're talking about one interview that produces a set of forms--I usually see the originals also listed together on one page that has the related forms). So I'm happy to defer, but we could open an issue to make it more flexible later.

Reply:
Makes sense, I hadn't thought of just linking to the single page (most of the examples I've seen us use go directly to the PDF). I'll close this for now, it's not urgent and will come back up if it's something we want